requirements.txt

```diff
@@ -0,0 +1,4 @@
+torch
+typer
+wandb
+chardet
```

Comment:
is chardet used?

Reply:
This was likely just an [issue with my conda env](https://stackoverflow.com/questions/76664986/unexpected-chardet-depedency-with-wandb-how-to-fix-and-why-does-it-happen;); removing that